src/objects/X3_0_Bushenyi_Maize__c.object

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
X3_0_ ???? doesnt look like good naming. I understand wanting to prefix, but X3_--- doesn't looks like poor naming